src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -446,12 +446,19 @@ private static string GetFixCategory(DiagnosticSeverity severity)
                     return null;
                 }
 
+                var sourceText = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+
+                var pastedTextSpanOpt =
+                    state.Target.Owner._pasteTrackingService.TryGetPastedTextSpan(sourceText.Container, out var pastedTextSpan)
+                    ? (TextSpan?)pastedTextSpan
+                    : null;
+
                 if (GlobalOptions.GetOption(EditorComponentOnOffOptions.CodeRefactorings) &&
                     state.Target.Owner._codeRefactoringService != null &&
                     state.Target.SubjectBuffer.SupportsRefactorings())
                 {
                     if (await state.Target.Owner._codeRefactoringService.HasRefactoringsAsync(
-                            document, selection.Value, cancellationToken).ConfigureAwait(false))
+                            document, selection.Value, pastedTextSpanOpt, cancellationToken).ConfigureAwait(false))
```

Comment:
so, generally, we've been trying to move to the model more where we have 'args' that can get passed along with the op, to avoid the 'just jam a ton of individual parameters along' .  Just food for thought.